File: Week1/mario.c
#include <stdio.h>
#include <cs50.h>

int main()
{
    int a;
    do{
        printf("Height: ");
        scanf("%d", &a);
    }while(a < 1 || a > 8);

    for(int i = 0; i < a; i++)
    {
        int spaces, hashes;
        spaces = a-i-1;
        hashes = i+1;
        for(int j = 0; j < spaces; j++){
            printf(" ");
        }
        for(int k = 0; k < hashes; k++){
            printf("#");
        }
        printf("  ");
        for(int k = 0; k < hashes; k++){
            printf("#");
        }
        printf("\n");
    }
}

File: Week6/sentimental-mario-more/mario.py
height = 0

# prompting user until right input
while (height < 1 or height > 8):
    try:
        height = int(input("Height: "))
    except ValueError:
        continue

# traversing through each level
for i in range(height):
    # number of leading spaces for each level
    spaces = height - i - 1

    # number of hashes for each level
    hashes = i + 1

    print(" " * spaces, end="")
    print("#" * hashes, end="  ")
    print("#" * hashes)